src/hooks/useScrollDisable.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from "react";
+
+const useScrollDisable = () => {
+  useEffect(() => {
+    const scrollY = window.scrollY;
+
+    if (scrollY) {
```

Comment:
```suggestion
    if (scrollY > 0) {
```

[`window.scrollY`에 대한 MDN 공식 문서](https://developer.mozilla.org/ko/docs/Web/API/Window/scrollY)
> scrollY는 현재 뷰포트 위쪽 모서리의 Y좌표를 반환하고, 뷰포트가 없으면 0을 반환합니다.

공식 문서에 명시된 것처럼 뷰포트가 없으면 0을 반환하기 때문에 더 명확하게 의미 파악이 될 수 있도록 조건이 _0보다 클 때_ 라고 수정하는건 어떨까요?!

Reply:
수정하였습니다 :)